draft-ietf-quic-version-negotiation.md

```diff
@@ -154,8 +155,8 @@ of Offered Versions (see {{server-fleet}}) in a Supported Version field. The
 server MAY add reserved versions (as defined in {{Section 6.3 of QUIC}}) in
 Supported Version fields.
 
-Clients will ignore a Version Negotiation packet if it contains the Original
-Version attempted by the client (see {{downgrade}}). The client also ignores a
+Clients MUST ignore a Version Negotiation packet if it contains the Original
+Version attempted by the client (see {{downgrade}}). The client MUST also ignore a
```

Comment:
Similarly, this was intentional to avoid repeating 8999

Reply:
See above. My problem is that naked statements of how the client behaves need to be supported by something. If you want to put (as required by...) that would be fine.